diff -upr openshot-1.3.0-orig/openshot/classes/av_formats.py openshot-1.3.0/openshot/classes/av_formats.py
--- openshot-1.3.0-orig/openshot/classes/av_formats.py	2011-01-29 13:58:49.000000000 -0600
+++ openshot-1.3.0/openshot/classes/av_formats.py	2011-02-14 14:45:31.313113270 -0600
@@ -36,7 +36,7 @@ from language import Language_Init
 
 class formats:
 	
-	def __init__(self, melt_command="melt"):
+	def __init__(self, melt_command="mlt-melt"):
 		# init melt command
 		self.melt_command = melt_command
 		
diff -upr openshot-1.3.0-orig/openshot/windows/preferences.py openshot-1.3.0/openshot/windows/preferences.py
--- openshot-1.3.0-orig/openshot/windows/preferences.py	2011-01-29 22:13:46.000000000 -0600
+++ openshot-1.3.0/openshot/windows/preferences.py	2011-02-14 14:46:33.354416506 -0600
@@ -329,7 +329,7 @@ class Settings:
 		"default_profile" : "DV NTSC",
 		"project_file_type" : "ascii",
 		"max_history_size" : "20",
-		"melt_command" : "melt",
+		"melt_command" : "mlt-melt",
 		"blender_command" : "blender",
 		"output_mode" : "sdl",
 		"use_stock_icons" : "Yes",
 # -------------------------------------
